libsrc/apple2/get_tv.s

```diff
@@ -0,0 +1,187 @@
+;
+; Colin Leroy-Mira <colin@colino.net>, 2025
+;
+; unsigned char __fastcall__ get_tv(void)
+;
+        .export         _get_tv
+
+        .import         _set_iigs_speed, _get_iigs_speed
+        .import         ostype
+
+        .constructor    calibrate_tv, 2
+
+        .include    "accelerator.inc"
+        .include    "apple2.inc"
+        .include    "get_tv.inc"
+
+; The only thing remaining from that code after init
+.proc _get_tv
+        lda     tv
+        ldx     #$00
```

Comment:
Please think about the cc65 policy to mark this as integer promotion. In the apple2 specific code base this is consistently doene by writing ldx >#$0000